purchase_order_weight_volume/README.rst

```diff
@@ -47,7 +47,7 @@ To use this module, you need to:
 * You will see order total weight and volume, tab "Other information".
 * From this tab, you can also choose whether to display weight and/or volume info in report.
 
-"Display Order Weight in PO" and "Display Order Weight in PO" can be enabled/disabled
+"Display Order Weight in PO" and "Display Order Volume in PO" can be enabled/disabled
```

Comment:
@tuantrantg there are some minor changes
- this part comes from module's [usage](https://github.com/tuantrantg/purchase-workflow/blob/16.0-mig-purchase_order_weight_volume/purchase_order_weight_volume/readme/USAGE.rst#L4), so ideally you should have updated that file
- then, you can use https://github.com/OCA/maintainer-tools#readme-generator to regenerate the readme as OCA standard

Reply:
a @dzungtran89 please review again, thank you